src/exceptions.h

```diff
@@ -32,6 +32,9 @@ class InternalErrorException : public HeraException {
 class VMTrap : public HeraException {
   using HeraException::HeraException;
 };
+class ArgumentOutOfRange : public HeraException {
```

Comment:
Is this intended to be used in OOB exceptions for the EEI?

Reply:
More like out of range for value types, such as `blockNumber` too high for `getBlockHash`, `gas` negative for `useGas`, etc.